.github/workflows/update.yaml

```diff
@@ -0,0 +1,30 @@
+name: Check for updates
```

Comment:
this is usually taken care by adding the proper stanza in the manifest.

Reply:
Not entirely sure what you're saying, this custom workflow is here to update future non-main branches, where each one is dedicated to a specific runtime version: 23.08, 24.08...
Can this task be accomplished by a manifest stanza?

Reply:
I knew this was the only way to have multiple builds of the same package for different runtime versions.

Reply:
nevermind.